website/static/js/nodeControl.js

```diff
@@ -221,6 +221,7 @@ var ProjectViewModel = function(data) {
      * Add project to the Project Organizer.
      */
     self.addToDashboard = function() {
+        $('#addDashboardFolder').tooltip('hide');
```

Comment:
@sloria  I don't think we need generalize this. Because the overlapping of tooltip only happens to the case in which two tooltip elements are controlled by if and ifnot binding.  To be more specific, those if binding can delete the DOM but can't delete the tooltip associated with the DOM. Therefore, the remained tooltip would overlap with the new tooltip.  In this case,  hide action is introduced before if binding delete the DOM.

So besides this use case, we don't need to hide tooltip when clicking.